cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,13 +24,107 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
+
+PRODUCT_FEATURES = [
+    [
+        (
+            [
+                "Everything",
+                "Settings",
+                "Configuration",
+                "Access Control",
+                "Tenants",
+                "Modify",
+                "Manage Quotas",
+                "Manage Quotas (My Company)",
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Settings",
+                "Configuration",
+                "Access Control",
+                "Tenants",
+                "Modify",
+                "Manage Quotas",
+                "Manage Quotas ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Add",
+                "Add ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Edit",
+                "Edit ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Delete",
+                "Delete ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Copy",
+                "Copy ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+]
+
 
 def events_check(updates=False):
```

Comment:
This function appears to be pretty useless without the BZ check. Whatever is calling it can select which list to use based on the arg passed here.

Please just remove the function, and update any callers.